### DISCUSSION TITLE
Case management

### DISCUSSION THREAD

**Discussion by isobelseacombe** [UKHomeOffice/design-system#338] [Ideas]
Lots of us are working on products where we keep records of a case or an individual over time and need to retain, view and manage large amounts of information. 

## Managing a case load
- A list of cases a person is responsible for managing 
- Allocating cases to be managed
- Searching for a case
See case list pattern on MoJ pattern library for inspiration 
https://design-patterns.service.justice.gov.uk/patterns/case-list/

### Activity 
- Progress
- Updates
- Latest information
- Audit trail of how the case has been managed

### Case history
- Scanning a case overview
- Extensive case history view
- Building a picture of a person or a case file
- View decisions made about the case
- Pulling in information for a case file from multiple data sources
### Case notes
- Free text vs. guided with hints and checkboxes
- Adding additional context to a submission of information (e.g an expiry date - add extra information alongside)

**Comment by MikeDThorpe** [comment]
### MLA Document Processing

MLA IDP (intelligent document processing) system uses a comparative pattern to the MOJ example above for managing active case loads across a team.

### Listing a case load with active statuses

The view in the below images lists the status of all uploaded documents that are in a queue for AWS Textract 'processing'. Once processed case files are 'ready' to upload for review. Status can also be 'locked' to show another case worker is working on it or if a senior account has locked the case for a particular reason.

![Screenshot 2022-02-23 at 09 14 33](https://user-images.githubusercontent.com/47759631/155292845-a969b0ef-b2a5-45cc-8f02-4a0f83ab1679.png)

![Screenshot 2022-02-23 at 09 14 53](https://user-images.githubusercontent.com/47759631/155293369-2f91343c-5527-4fd1-b251-9bbbc98f52cc.png)

### Handling priority case loads

Once a document has been reviewed and submitted to the case working system it is removed from the queue list. Case workers are then able to continue with the next 'highest priority' case file without returning to the the queue list.

![Screenshot 2022-02-23 at 09 26 06](https://user-images.githubusercontent.com/47759631/155294372-9a619a92-8347-4ab6-b4e1-04e9f219cbda.png)

Case workers can also open the highest priority case straight from the homepage.

![Screenshot 2022-02-23 at 09 45 07](https://user-images.githubusercontent.com/47759631/155294953-5797ea00-81cc-4569-84ad-3520de28f28a.png)
